### ISSUE TITLE
How can I load extensions

### ISSUE THREAD

**Issue by amexboy** [liquibase/liquibase-groovy-dsl#33]
Hi

I see the examples has loadData, but liquibase complains `'loadData' is not a valid element of a ChangeSet` 

How can load extensions?

**Comment by stevesaliman** [comment]
`loadData` is part of the Liquibase core, so you shouldn't need to load any extensions.  How are you using `loadData` in your change set?  It requires a map and a closure.  If one of them is missing, you'd get an error like the one you are seeing.  Here is an example of a valid `loadData` element:
```groovy
changeSet (id: 'myChangeSet', author: 'me') {
  loadData(file: 'data.csv', tableName: 'myTable') {
    column(name: 'id')
    column(name: 'description')
  }
}
```

**Comment by amexboy** [comment]
@stevesaliman I didn't know the closure was required. Even an empty closure works. 

The xml counter part takes identityInsertEnabled?

**Comment by amexboy** [comment]
@stevesaliman identityInsertEnabled is part of an extension (liquibase-mssql). I couldn't figure out how to load that. I even tried to create a new extension, but it is not being detected, any clue?

**Comment by amexboy** [comment]
@stevesaliman I just figured out that the groovy-dsl doesn't detect extensions and I made my modifications to enable that. 

https://github.com/liquibase/liquibase-groovy-dsl/pull/36

Sent a pull request

**Comment by stevesaliman** [comment]
I just released version 2.0.2 of the Groovy DSL with the changes that allow extension provided changes to be loaded.  Thank you for all of your help on this issue.

**Comment by amexboy** [comment]
Glad I could help.


Amanuel Nega (Software Engineer)
